tools/c7n_gcp/tests/data/flights/bucket-iam-policy/get-storage-v1-b-cloud-custodian-test-1-iam_1.json

```diff
@@ -0,0 +1,55 @@
+{
+  "headers": {
+    "etag": "CAI=",
+    "content-type": "application/json; charset=UTF-8",
+    "date": "Thu, 17 Jun 2021 21:11:59 GMT",
+    "vary": "Origin, X-Origin",
+    "cache-control": "private, max-age=0, must-revalidate, no-transform",
+    "expires": "Thu, 17 Jun 2021 21:11:59 GMT",
+    "content-length": "849",
+    "server": "UploadServer",
+    "alt-svc": "h3=\":443\"; ma=2592000,h3-29=\":443\"; ma=2592000,h3-T051=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"",
+    "status": "200",
+    "content-location": "https://storage.googleapis.com/storage/v1/b/cloud-custodian-test-1/iam?alt=json"
+  },
+  "body": {
+    "kind": "storage#policy",
+    "resourceId": "projects/qa-onprem/buckets/cloud-custodian-test-1",
```

Comment:
Can we go through and use mock data here? It could be something like `projects/custodian/buckets/bucket-1`

Reply:
Honestly maybe a find and replace of `qa-onprem` with `custodian` would do the trick